docs/quickstart.md

```diff
@@ -23,12 +23,40 @@ $ helm init
 ```
 
 ### Install and configure Draft
-- Download the Draft binary using Homebrew via the commands below or from the [official releases page](https://github.com/Azure/draft/releases)
+
+#### Standalone Binary
+
+Download the latest release of Draft from the [latest releases page](https://github.com/Azure/draft/releases/latest), unpack the binary and place it somewhere on your $PATH.
+
```

Comment:
Perhaps add instructions on how to unpack it for linux systems? I wasn't able to get GoFish working on my machine, using Ubuntu for Windows.

1. Download the binary
2. Unpack it ( tar -zxvf draft-v0.14.1-linux-amd64.tar.gz )
3. Add it to path ( mv linux-amd64/draft /usr/local/bin/draft )

Reply:
That's a good suggestion. Thanks @cnadolny! I'll add those steps here.

Hmm, I have not tested GoFish under Windows Subsystem for Linux yet, but the native Windows client should work fine via https://www.gofi.sh/#install. If you have an issue I'd love to hear it at https://github.com/fishworks/gofish! Thanks for trying it out. :)

Reply:
added. Thanks!